autopts/wid/mcp.py

```diff
@@ -341,24 +350,29 @@ def hdl_wid_20110(params: WIDParams):
     addr_type = btp.pts_addr_type_get()
     addr = btp.pts_addr_get()
 
-    if "0x00F3" in params.description:
+    """ PTS is providing handle value instead of expected action so we map it here based on
+    test case name
+    """
+    if 'MCP/CL/CGGIT/CHA/BV-07-C' == params.test_case_name:
         btp.mcp_track_position_set(100, addr_type, addr)
         ev = stack.mcp.wait_track_position_ev(addr_type, addr, 10)
-    elif "0x00F6" in params.description:
+    elif 'MCP/CL/CGGIT/CHA/BV-08-C' == params.test_case_name:
         btp.mcp_playback_speed_set(64, addr_type, addr)
         ev = stack.mcp.wait_playback_speed_ev(addr_type, addr, 10)
-    elif "0x0101" in params.description:
+    elif 'MCP/CL/CGGIT/CHA/BV-11-C' == params.test_case_name:
+        btp.mcp_current_track_obj_id_set(16777499, addr_type, addr)
```

Comment:
Where does the literal ints (e.g. `16777499`) come from?

Reply:
from what I see those are hardcoded value returned by PTS....  I'll see if we can track this and avoid hardcoded values